src/events/crawler/scrapers/USA/KS/index.js

```diff
@@ -0,0 +1,176 @@
+  _baseUrl: 'https://khap2.kdhe.state.ks.us/NewsRelease/COVID19/',
+  source: {
+    name: 'Kansas Department of Health and Environment',
+    url: 'https://govstatus.egov.com/coronavirus'
+  },
+  maintainers: [maintainers.paulboal],
+  _countyMap: {},
+  _counties: [
+    'Allen County',
+    'Anderson County',
+    'Atchison County',
+    'Barber County',
+    'Barton County',
+    'Bourbon County',
+    'Brown County',
+    'Butler County',
+    'Chase County',
+    'Chautauqua County',
+    'Cherokee County',
+    'Cheyenne County',
+    'Clark County',
+    'Clay County',
+    'Cloud County',
+    'Coffey County',
+    'Comanche County',
+    'Cowley County',
+    'Crawford County',
+    'Decatur County',
+    'Dickinson County',
+    'Doniphan County',
+    'Douglas County',
+    'Edwards County',
+    'Elk County',
+    'Ellis County',
+    'Ellsworth County',
+    'Finney County',
+    'Ford County',
+    'Franklin County',
+    'Geary County',
+    'Gove County',
+    'Graham County',
+    'Grant County',
+    'Gray County',
+    'Greeley County',
+    'Greenwood County',
+    'Hamilton County',
+    'Harper County',
+    'Harvey County',
+    'Haskell County',
+    'Hodgeman County',
+    'Jackson County',
+    'Jefferson County',
+    'Jewell County',
+    'Johnson County',
+    'Kearny County',
+    'Kingman County',
+    'Kiowa County',
+    'Labette County',
+    'Lane County',
+    'Leavenworth County',
+    'Lincoln County',
+    'Linn County',
+    'Logan County',
+    'Lyon County',
+    'Marion County',
+    'Marshall County',
+    'McPherson County',
+    'Meade County',
+    'Miami County',
+    'Mitchell County',
+    'Montgomery County',
+    'Morris County',
+    'Morton County',
+    'Nemaha County',
+    'Neosho County',
+    'Ness County',
+    'Norton County',
+    'Osage County',
+    'Osborne County',
+    'Ottawa County',
+    'Pawnee County',
+    'Phillips County',
+    'Pottawatomie County',
+    'Pratt County',
+    'Rawlins County',
+    'Reno County',
+    'Republic County',
+    'Rice County',
+    'Riley County',
+    'Rooks County',
+    'Rush County',
+    'Russell County',
+    'Saline County',
+    'Scott County',
+    'Sedgwick County',
+    'Seward County',
+    'Shawnee County',
+    'Sheridan County',
+    'Sherman County',
+    'Smith County',
+    'Stafford County',
+    'Stanton County',
+    'Stevens County',
+    'Sumner County',
+    'Thomas County',
+    'Trego County',
+    'Wabaunsee County',
+    'Wallace County',
+    'Washington County',
+    'Wichita County',
+    'Wilson County',
+    'Woodson County',
+    'Wyandotte County'
+  ],
+  scraper: {
+    '2020-03-18': async function() {
+      const date = process.env.SCRAPE_DATE || datetime.getYYYYMMDD();
+      const datePart = datetime.getMonthDYYYY(date);
+      this.url = `${this._baseUrl}COVID-19_${datePart}_.pdf`;
+
+      let rows = null;
+      try {
+        rows = await fetch.pdf(this.url, date, { alwaysRun: true });
```

Comment:
The PDF scraping logic was changed recently to allow us to scrape maps, so your code will need a slight change. Could you apply the same logic as https://github.com/lazd/coronadatascraper/blob/master/src/events/crawler/scrapers/USA/MA/index.js#L51L57

Reply:
Update made and I also had to make a fix to Kansas posting a table with 2 or 4 columns.  Please review again.